### PR TITLE
Added Voices of the ElePHPant

### PR DESCRIPTION
**Pull request by calevans**
I apologize for the multiple PRs. I am really not that inept. :) I have added to the list my podcast, Voices of the ElePHPant and a youtube channel I manage. The Nomad PHP Lightning Talk channel is where we post our monthly lightning talks and it is available to everyone. 

Cheers!
=C=

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -915,7 +915,7 @@ Various resources, such as books, websites and articles, for improving your PHP
 * [PHP UK Conference](https://www.youtube.com/user/phpukconference/videos) - A collection of videos from the PHP UK Conference.
 * [Programming with Anthony](https://www.youtube.com/playlist?list=PLM-218uGSX3DQ3KsB5NJnuOqPqc5CW2kW) - A video series by Anthony Ferrara.
 * [Taking PHP Seriously](https://www.infoq.com/presentations/php-history) - A talk outlining PHP's strengths by Keith Adams of Facebook.
-* [Voices of the ElePHPant](https://voicesoftheelephpant.com/)
+* [Voices of the ElePHPant](https://voicesoftheelephpant.com/) Interviews with teh people that make the PHP community speical.
```
Comment (josegonzalez) [inline README.md]: *the*

Comment (calevans) [inline README.md]: d'Oh

File: README.md [hunk 2]
```diff
@@ -915,7 +915,7 @@ Various resources, such as books, websites and articles, for improving your PHP
 * [PHP UK Conference](https://www.youtube.com/user/phpukconference/videos) - A collection of videos from the PHP UK Conference.
 * [Programming with Anthony](https://www.youtube.com/playlist?list=PLM-218uGSX3DQ3KsB5NJnuOqPqc5CW2kW) - A video series by Anthony Ferrara.
 * [Taking PHP Seriously](https://www.infoq.com/presentations/php-history) - A talk outlining PHP's strengths by Keith Adams of Facebook.
-* [Voices of the ElePHPant](https://voicesoftheelephpant.com/) Interviews with teh people that make the PHP community speical.
+* [Voices of the ElePHPant](https://voicesoftheelephpant.com/) Interviews with the people that make the PHP community speical.
```
Comment (josegonzalez) [inline README.md]: Oh, also `special`.

**Comment by josegonzalez** [comment]
Thanks for the pr!

**Pull request merged**